server/lib/nl_page_config.py

```diff
@@ -350,6 +350,39 @@ def build_page_config(detection: Detection, data_spec: DataSpec):
       category.stat_var_spec[sv_key].name = sv2name[primary_sv]
       category.stat_var_spec[sv_key].denom = "Count_Person"
 
+  # Render scatter plot if query asks for a correlation
+  # IMPORTANT: Right now this is fragile, as it will only
+  # use SVs from the most recent context.
+  # Also assumes that
+  #    - previous context had a primary_sv that was not "".
+  #    - data_spec.selected_svs is not empty
+  elif classificationType == ClassificationType.CORRELATION:
+
+    # get first stat var from previous context
+    sv_1 = data_spec.context_sv
+    # get second stat var from current
+    sv_2 = data_spec.selected_svs[0]
+    sv_names = get_sv_name([sv_1, sv_2])
+
+    sv_1_key = sv_1 + "_scatter"
+    sv_1_name = sv_names[sv_1]
```

Comment:
also for future - the key's don't need a prefix. the keys can be reused across tiles.